### ISSUE TITLE
Updating to latest pg.v5 results in TLS issues

### ISSUE THREAD

**Issue by jbeemster** [go-pg/pg#451]
On upgrading to the latest version of pg.v5 we have run into issues with TLS.  In short all my connections are ended with the following error message:

```bash
read: connection reset by peer
```

Downgrading to pg.v4 has resolved these issues.

Using the following TLS Config in v4 works but fails in v5:

```go
tlsConfig = &tls.Config{
    InsecureSkipVerify: true,
}
```

This is just the default tlsConfig.

Any insight into how we could fix the issue would be greatly appreciated!

**Comment by vmihailenco** [comment]
Fixed in v5.2.3

**Comment by jbeemster** [comment]
Thanks @vmihailenco !
